UnitTests/fixtures/LocStructuredOnly.strings

```diff
@@ -0,0 +1,3 @@
+"settings.navigation-bar.title.even.deeper" = "Settings";
+"SETTINGS.USER_PROFILE_SECTION.HEADER_TITLE" = "User Profile Settings";
+"seTTings.uSer-proFile-sectioN.footer_text" = "Here you can change some user profile settings.";
```

Comment:
Also wondering what happens if a structure happens to be both a leaf and a node of the structure, like:

```
"settings.title" = "Titre of section";
"settings.title.details" = "Subtitle";
```
In that case wouldn't there be some conflict like inside `enum SettingsL10n` you'd have both an case for `title` and a subenum for `title` containing `detail`?

I guess that will work ok because if I understand correctly, the names inside `enum SettingsL10n` will be `case title` and `enum TitleL10n` so they won't conflict, but it would be nice to add such a use case in that `LocStructuredOnly.strings` file to be sure we handle it properly without conflicts.